src/components/SituationFinish/styled.ts

```diff
@@ -49,19 +54,77 @@ export const ButtonContainer = styled.div`
   gap: 0.75rem;
 `;
 
-export const Button = styled.button`
+export const HandleNext = styled.div`
```

Comment:
```suggestion
export const HomeLink = styled(Link)`
```
여기 혹시 `Link` 태그 사용하지 않고 `div`태그에 `onClick`으로 메인 페이지로 이동하신 이유가 있으신가요??

Reply:
다시하기를 새로고침으로 구현하려고 하려는게 이유였는데 새로고침은 컴포넌트를 새로 렌더링해서 여러 상태들이 보존되지 않는다고 하네요 그래서 Link로 변경하고 /situation 으로 경로 이동시키는 방법으로 변경했습니다 !